rasterio/__init__.py

```diff
@@ -14,13 +14,13 @@ def emit(self, record):
             pass
 
 from rasterio._base import gdal_version
-from rasterio._copy import copy
+from rasterio.shutil import copy as _copy  # This needs to be removed but triggers a wild error: ImportError: cannot import name 'WindowMethodsMixin'
```

Comment:
@sgillies I really have no idea what causes this, but I confirmed its also a problem on `master` and opened https://github.com/mapbox/rasterio/issues/1174.

Reply:
No harm in working around in for this PR.

Reply:
This is resolved by #1177.